src/wp-content/themes/twentytwentythree/style.css

```
@@ -0,0 +1,16 @@
+/*
+Theme Name: Twenty Twenty-Three
+Theme URI: https://github.com/WordPress/twentytwentythree
+Author: wordpressdotorg
```

This should be `the WordPress team` to match the other default themes.

```suggestion
Author: the WordPress team
```